make_ghpages/mod/templates/main_index.html

```diff
@@ -29,10 +29,14 @@ <h1>
         {% for plugin_name, plugin in plugins.iteritems() %}
         <div class='submenu-entry'>
         <h2><a href="{{ plugin.subpage }}">{{ plugin_name}}: {{ plugin.name }} package</a></h2>
-        {% if plugin.setupinfo.description %}
-        <p class=description>{{plugin.setupinfo.description}}</p>
+        {% if plugin.setup_json.description %}
+        <p class=description>{{plugin.setup_json.description}}</p>
         {% endif %}
-        <p class="currentstate">Current state: {{ plugin.state }}</p>
+        <p class="currentstate">Current state: {{ plugin.state }}
```

Comment:
Could you avoid the space before the comma below? There are tricks in jinja2 to avoid unwanted spacing, see e.g. http://jinja.pocoo.org/docs/2.10/templates/#whitespace-control

Reply:
fixed